src/pages/registration/FilesAndLicensePanel.tsx

```diff
@@ -74,8 +74,10 @@ const FilesAndLicensePanel: FC<FilesAndLicensePanelProps> = ({ uppy }) => {
         {({ name, remove, push }: FieldArrayRenderProps) => (
           <>
             {files.length > 0 && (
-              <StyledUploadedFiles>
-                <Typography variant="h2">{t('files_and_license.files')}</Typography>
+              <StyledBackgroundDiv backgroundColor={theme.palette.section.dark}>
+                <Typography color="primary" variant="h2">
```

Comment:
kommer til å bli svimmel av dette. i theme har vi definert `primary` til å være blå, da vil jeg forvente at fargen er samme blåfarge som er definert uansett hvor jeg bruker `color="primary"`. Men hvis `StyledBackgroundDiv` er parent (kan være flere nivå lenger opp) så kan `primary` være en helt annen farge (hvit i dette tilfellet) :/

Reply:
Ja? Det er jo det som er poenget? 😅

du mener at du forventer at fargen skal være svart når du skriver color=«primary»?

Reply:
Eller mener du at vi burde ha forskjellige komponenter til forskjellige MUI-komponenter? Slik at vi har en for typography som bruker textPrimary?

Reply:
Uansett så kommer jeg til dette nå straks 😅 husk på at dette fortsatt ikke er ferdig nok til å rulles ut til test. Men må ta en ting om gangen.

Reply:
jeg forventer at "primary" alltid gir blåfargen vi har satt som `primary.main` i theme.
hvis vi finner ut at denne StyledBackgroundDiv skal erstattes med en enkel div blir jo fargen blå også, så hierarkiet bestemmer egentlig fargen mer enn `color` propen atm
Hvis jeg hadde kommet inn som ny utvikler hadde det i hvert fall vært veldig forvirrende
Hva jeg mener vi burde gjøre? Har ikke peiling 🙃